packages/cli/src/commands/releases/info.ts

```diff
@@ -0,0 +1,59 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import * as shellescape from 'shell-escape'
+const {forEach} = require('lodash')
+
+import {findByLatestOrId} from '../../lib/releases/releases'
+import {description, color as getStatusColor} from '../../lib/releases/status_helper'
+
+export default class Info extends Command {
+    static topic = 'releases';
+    static description = 'view detailed information for a release';
+    static flags = {
+      json: flags.boolean({description: 'output in json format'}),
+      shell: flags.boolean({char: 's', description: 'output in shell format'}),
+      remote: flags.remote(),
+      app: flags.app({required: true}),
+    };
+
+    static args = {
+      release: Args.string(),
+    };
+
+    public async run(): Promise<void> {
+      const {flags, args} = await this.parse(Info)
+      const {json, shell, app} = flags
+      const release = await findByLatestOrId(this.heroku, app, args.release)
+
+      const {body: config} = await this.heroku.get<Heroku.ConfigVars>(`/apps/${app}/releases/${release.version}/config-vars`)
```

Comment:
Since `config` is conditionally referenced in the `else` block on line 34, would it make sense to initialize it there?

Reply:
Good call, thanks